ethereum/.env.optimism.testnet

```diff
@@ -7,3 +7,12 @@ INIT_CHAIN_ID=24
 INIT_GOV_CHAIN_ID=0x1
 INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
 INIT_EVM_CHAIN_ID=420
+
+# Bridge Migrations
+BRIDGE_INIT_CHAIN_ID=24
+BRIDGE_INIT_GOV_CHAIN_ID=0x1
+BRIDGE_INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
+# Not sure if there should be a WETH, confirm this before mainnet deploy!
+BRIDGE_INIT_WETH=0x4200000000000000000000000000000000000006
```

Comment:
I think this should be all zeros. 0x4200000000000000000000000000000000000006 is bridged WETH and we don't want to double wrap it.

Reply:
Fixed